<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>IISER Kolkata | Description</title>

    <link rel="stylesheet" href="../old two/main.css">
    <link rel="stylesheet" href="./engineering.css">
    <link rel="stylesheet" href="../old two/katex/katex.min.css">
    <link rel="stylesheet" href="../old two/grassAnimation.css">
    <link rel="stylesheet" href="../old two/hamburger.css">
    <link rel="stylesheet" href="../colorpalatte.css">


    <script src="../old two/katex/katex.min.js"></script>
    <script src="../old two/katex/contrib/auto-render.min.js"></script>
    <script src="../old two/katex/contrib/mhchem.min.js"></script>

    <script src="../old two/js/jquery.min.js"></script>
    <script src="../old two/js/jquery.rwdImageMaps_lq5sye.js"></script>
    <script src='../old two/js/sketch.min.js'></script>

</head>

<body>


    <header class="nav_Container" id="nav_Container">
        <nav class="menu--right" role="navigation">
            <div class="menuToggle">
                <input type="checkbox" onclick="hide()" />
                <span class="" style="background-color: #FFC000" id="span1"></span>
                <span class="" style="background-color: #FFC000" id="span2"></span>
                <span class="" style="background-color: #FFC000" id="span3"></span>
                <div class="menuItem">
                    <ul class="clear_ul">
                        <li><a href="#">PROJECT</a></li>
                        <li><a href="#">WET LAB</a></li>
                        <li><a href="#">DRY LAB</a></li>
                        <li><a href="#">HUMAN PRACTICES</a></li>
                        <li><a href="#">TEAM</a></li>
                    </ul>
                </div>
            </div>
        </nav>
        <div class="dropdown hidden" id="dropdown">
            <ul class="clear_ul" id="dropDown_ul">
            </ul>
        </div>
    </header>

    <!-- <div class="" id="behind">
        <h1 class="pgintro">
            ENGINEERING
        </h1>
        <div class="sun"></div>
        <canvas id="test"></canvas>
    </div> -->

    <div class="" style="background-image: url(../3.png);height: 100vh;width: 100%vw">
        <img src="../thumbnails/3.png" alt="" style="width: 100vw;height: 100vh;">
    </div>

    <br><br>

    <h1 class="separator_txt" style="text-align: center;font-size: 4rem;">
        Cycles
    </h1>
    <div class="section_button_container">
        <div class="section_button" style="cursor: pointer;" onclick="window.location.href = './#1'">
            <a href="#1">
                XOO CULTURE SYSTEM
            </a>
        </div>
        <div class="section_button" style="cursor: pointer;" onclick="window.location.href = './#2'">
            <a href="#2">
                OMV Isolation and Purification
            </a>
        </div>
        <div class="section_button" style="cursor: pointer;" onclick="window.location.href = './#3'">
            <a href="#3">
                TAT genetic circuit
            </a>
        </div>
        <div class="section_button" style="cursor: pointer;" onclick="window.location.href = './#4'">
            <a href="#4">
                OMV Fusion Verification with Xanthomonas
            </a>
        </div>
        <div class="section_button" style="cursor: pointer;" onclick="window.location.href = './#5'">
            <a href="#5">
                Machine Learning Model for Mutational Response
            </a>
        </div>
    </div>

    <br><br>
    <!-- ---------------------------------- -->
    <div class="separator" id="1">
        <h1 class="separator_txt">
            XOO CULTURE SYSTEM
        </h1>
    </div>

    <div class="superStickyContainer">
        <div class="sticky_index_container"
            style="width: 10%;display: flex;flex-direction: column;align-items: center;">
            <div class="sticky_index2">
                <span>XOO CULTURE SYSTEM</span>
            </div>
            <div></div>
        </div>
        <div class="content_container" style="width: 90%;">
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Design
                        </h1>
                        <img src="./Design.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Design
                        </h1>
                        <p class="p_content">
                            Our team embarked on an exciting journey to study the growth of two strains of Xanthomonas
                            bacteria: Xanthomonas oryzae pv oryzae BXO43 from the Center for DNA Fingerprinting and
                            Diagnostics (CDFD) and Xanthomonas oryzae pv. oryzae BXO1 from the Center for Cellular and
                            Molecular Biology (CCMB). The aim was to optimize their growth conditions and conduct
                            various assays to gain valuable insights.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Build
                        </h1>
                        <img src="./Build.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Build
                        </h1>

                        <p class="p_content">
                            Our first task was to create a suitable environment for these bacterial strains. We obtained
                            glycerol
                            stocks of Xanthomonas oryzae pv. oryzae BXO43 from CDFD. To facilitate their growth, we
                            prepared
                            rifampicin PSA plates and covered them with aluminum foil, graciously borrowed from Dr.
                            Sreeramaiah
                            Gangappa's Plant Biology lab.
                        </p>
                        <p class="p_content">

                            We streaked Xanthomonas oryzae pv. oryzae BXO43 on two rifampicin PSA plates and placed them
                            in a
                            28°C
                            incubator for a week. Simultaneously, we received agar stabs of Xanthomonas oryzae pv.
                            oryzae BXO1
                            from
                            CCMB. These were streaked onto PSA plates without any antibiotics and also incubated at
                            28°C.
                        </p>
                        <p class="p_content">

                            To provide the necessary growth medium, we attempted to prepare 100ml of PSA rich media
                            three times,
                            but
                            each time, it got contaminated. Despite the setbacks, we persevered.
                        </p>
                        <p class="p_content">

                            After seven days, we observed the growth of Xanthomonas BXO43 colonies . Unfortunately, one
                            of the
                            plates that originated from the rifampicin-containing colonies was contaminated. To rectify
                            this, we
                            streaked two LB agar plates from the remaining colonies and placed them in the 28°C
                            incubator.
                        </p>
                        <p class="p_content">

                            In parallel, we initiated primary cultures of Xanthomonas BXO1 from CCMB and kept them in an
                            incubator
                            shaker at 28°C for 24 hours. We encountered some complications due to issues with agar and
                            LB
                            concentrations from the previous day, but we persisted in our efforts.
                        </p>
                        <p class="p_content">

                            One of the primary cultures showed turbidity after 24 hours, indicating growth, but the
                            other one
                            showed
                            no signs of development. We also measured the optical density (OD) of the primary culture,
                            which had
                            reached approximately 0.3.
                        </p>
                        <p class="p_content">

                            For further experimentation, we prepared three different growth media: LB, PS, and Nutrient
                            broth,
                            and
                            autoclaved them. Subsequently, we initiated primary cultures of Xanthomonas BXO43 from CCMB
                            in these
                            three different media, originating from two different Xanthomonas-streaked plates, resulting
                            in a
                            total
                            of six primary cultures. These were placed in the 28°C incubator at 250rpm.
                        </p>

                        <div class="table" id="engineeringSuccess_pt1"></div>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Test
                        </h1>
                        <img src="./Test.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Test
                        </h1>
                        <p class="p_content">
                            With our cultures in place, we began our series of tests and assays. We streaked Xanthomonas
                            oryzae BXO43 and BXO1 from previously streaked plates and closely monitored their growth at
                            various temperatures and time points to optimize their growth conditions.
                        </p>

                        <p class="p_content">
                            We also initiated a growth assay of the primary culture of Xanthomonas BXO1 in LB media at
                            28°C and 250rpm, recording their progress at different time intervals.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Learn
                        </h1>
                        <img src="./Learn.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Learn
                        </h1>
                        <p class="p_content">
                            As our experiments continued, we gathered valuable data on the growth patterns of
                            Xanthomonas oryzae BXO43 and BXO1 under various conditions. We learned about the challenges
                            of media contamination and the importance of meticulously maintaining culture conditions.
                        </p>
                        <p class="p_content">
                            Through our assays, we gained insights into the optimal growth conditions for Xanthomonas
                            BXO43 and laid the foundation for further research and experiments aimed at better
                            understanding these bacterial strains.
                        </p>
                        <div class="table" id="engineeringSuccess_pt2"></div>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/cropped-xoo-growth-curve.png"
                            alt="">
                    </div>
                </div>
            </div>
        </div>
    </div>
    <!-- ---------------------------------- -->

    <div class="separator" id="2">
        <h1 class="separator_txt">
            OMV Isolation and Purification
        </h1>
    </div>
    <div class="superStickyContainer">
        <div class="sticky_index_container"
            style="width: 10%;width: 10%;display: flex;flex-direction: column;align-items: center;">
            <div class="sticky_index2">
                <span>OMV Isolation and Purification</span>
            </div>
            <div></div>
        </div>
        <div class="content_container" style="width: 90%;">
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Design
                        </h1>
                        <img src="./Design.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Design
                        </h1>
                        </p>
                        <p class="p_content">
                            In the early stages of our project, we recognized that OMV (Outer Membrane Vesicle)
                            isolation was the foundational step we needed to master. With a clear goal in mind, we
                            embarked on a quest to find the most effective procedures for OMV isolation.
                        </p>
                        <p class="p_content">
                            To guide us in this endeavor, we drew inspiration from the previous year's iGEM IISER TVM
                            Team in 2022, who had worked extensively with hyper vesiculating strains as part of their
                            project. We reached out to a former member of that team, Sneha PR, who kindly agreed to
                            mentor us and help us obtain the hyper vesiculating E. coli strain we needed.
                        </p>
                        <p class="p_content">
                            Simultaneously, we scoured the academic landscape to identify professors with expertise in
                            hyper vesiculating E. coli strains and OMV isolation. Thanks to our Principal Investigator
                            (PI), we reached out to these professors via email. Fortunately, our efforts bore fruit, as
                            we secured assistance from several professors and obtained strains from their laboratories.
                        </p>
                        <p class="p_content">
                            Our next step was to delve into the study of these strains and compare their Outer Membrane
                            Vesicles. To do this, we needed a robust protocol for OMV isolation and purification.
                            Seeking guidance, we met with Dr. Amirul Islam Mallick, who offered valuable insights into
                            potential characterization studies we could undertake for OMVs
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Build
                        </h1>
                        <img src="./Build.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Build
                        </h1>
                        <p class="p_content">
                            With guidance from Afruja, a PhD student from Dr. Amirul Islam Mallick's lab, we began the
                            process of isolating OMVs.
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/amirul.jpg" alt="">

                        <h2 class="h2_content">For Bacterial Secondary Culture:</h2>
                        <ol class="p_content">
                            <li class="p_content">Prepare 250ml of bacterial secondary culture and place 4-6 autoclaved
                                Oakridge tubes
                                inside the laminar hood.</li>
                            <li class="p_content">Distribute the 250ml secondary culture equally among the 4-6 Oakridge
                                tubes.</li>
                            <li class="p_content">Centrifuge the Oakridge tubes at 8000 rpm at 4°C for 30 minutes to
                                pellet down the
                                bacteria.</li>
                            <li class="p_content">Perform membrane filtration on the supernatant using a 0.22-micron
                                filter.</li>
                            <li class="p_content">Collect the filtered supernatant into an autoclaved conical flask for
                                ultracentrifugation.</li>
                            <li class="p_content">Be cautious with the polycarbonate tubes, which are not autoclavable;
                                wash them with
                                distilled water.</li>
                            <li class="p_content">Fill the polycarbonate tubes up to the mark, ensuring approximately
                                90% capacity to
                                prevent tube breakage during ultracentrifugation.</li>
                            <li class="p_content">Weigh the tubes and their caps to ensure equal weights up to two
                                decimal points.</li>
                            <li class="p_content">Repeat the weighing procedure three times for verification.</li>
                            <li class="p_content">Perform ultracentrifugation at 25000 rpm, 4°C, for 4 hours.</li>
                            <li class="p_content">Once the centrifugation is complete, reduce the rpm speed gradually
                                and release the
                                vacuum.</li>
                            <li class="p_content">Gently open the lid and retrieve the rotor with the cassettes
                                attached.</li>
                            <li class="p_content">Discard the supernatant from each of the 6 tubes.</li>
                            <li class="p_content">Add 0.5-1 ml of 1X PBS buffer to one tube and pipette up and down to
                                dissolve the OMVs
                                into the buffer.</li>
                            <li class="p_content">Transfer the OMV-dissolved PBS buffer to the next tube.</li>
                            <li class="p_content">Rinse with 0.5-1 ml of 1X PBS buffer and collect in another tube.</li>
                            <li class="p_content">Store the MCTs at -20°C.</li>
                        </ol>


                        <h2 class="h2_content">For OMV Purification:</h2>

                        <ol class="p_content">
                            <li class="p_content">Thaw the stored cells at 4°C.</li>
                            <li class="p_content">Use a hood to balance the chilled SW41 tubes with a maximum of 10ml of
                                sample.</li>
                            <li class="p_content">Utilize the ultracentrifuge at 30krpm for 3 hours at 4°C.</li>
                            <li class="p_content">Discard the supernatant to reveal a transparent pellet.</li>
                            <li class="p_content">Dissolve the pellet in 300-500ul PBS.</li>
                            <li class="p_content">Use a sterile 0.45um / 0.22um syringe filter to filter the pellet.
                            </li>
                            <li class="p_content">Store the purified OMVs in a -20/-80°C refrigerator.</li>
                        </ol>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Test
                        </h1>
                        <img src="./Test.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Test
                        </h1>
                        <p class="p_content">
                            Armed with our isolated OMVs, we proceeded to conduct various tests and characterizations.
                            We employed scanning electron microscopy (SEM) and transmission electron microscopy (TEM) to
                            explore the OMVs' morphology and structure. These tests yielded successful results, allowing
                            us to gain deeper insights into our OMVs.
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/cropped-grid-26324.png"
                            alt="" srcset="">
                        <p style="text-align: center;">
                            <i>
                                TEM image of E.coli Δnlp strain
                            </i>
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/cropped-3-04.png" alt=""
                            srcset="">
                        <p style="text-align: center;">
                            <i>
                                SEM image
                            </i>
                        </p>
                        <p class="p_content">
                            In addition to SEM and TEM, we also performed dynamic light scattering (DLS) to determine
                            the size of the OMVs, further enriching our understanding of these crucial structures.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Learn
                        </h1>
                        <img src="./Learn.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Learn
                        </h1>
                        <p class="p_content">
                            Through meticulous planning, collaboration with mentors and professors, and hands-on
                            experimentation, we successfully achieved OMV isolation and purification. Our journey in the
                            world of OMVs had just begun, and we had gained valuable knowledge about the procedures and
                            characterization techniques involved.
                        </p>
                        <p class="p_content">
                            This phase of our project had set a strong foundation for the subsequent stages, where we
                            would delve deeper into the study of OMVs and their potential applications. With each step,
                            we continued to learn, adapt, and refine our approach, eager to unravel the mysteries of
                            these tiny vesicles and their significance in the field of synthetic biology.
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>
    <!-- ---------------------------------- -->

    <div class="separator" id="3">
        <h1 class="separator_txt">
            TAT genetic circuit
        </h1>
    </div>
    <div class="superStickyContainer">
        <div class="sticky_index_container"
            style="width: 10%;width: 10%;display: flex;flex-direction: column;align-items: center;">
            <div class="sticky_index2">
                TAT genetic circuit
            </div>
            <div></div>
        </div>
        <div class="content_container" style="width: 90%;">
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Design
                        </h1>
                        <img src="./Design.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Design
                        </h1>
                        <p class="p_content">
                            It all started when we stumbled upon a fascinating review article by Tracy Palmer titled
                            "Targeting of proteins to the twin-arginine translocation pathway." In this intricate world
                            of protein transport, we uncovered the Twin-Arginine Translocation pathway. Excited and
                            intrigued by its potential, we began to explore the possibility of utilizing this pathway
                            for our protein of interest: fnCas12a. Our goal was to transport fnCas12a to the periplasm,
                            a crucial step in our project.
                        </p>
                        <p class="p_content">
                            Our research unveiled the Tat ABCE complex as the ideal candidate for this ambitious
                            endeavor. To dive deeper, we turned to Uniprot, where we unearthed valuable insights into
                            Tat A, Tat B, Tat C, and Tat E. Armed with knowledge about their sequences, structures, and
                            functions, we were ready to embark on the journey of designing a genetic circuit that would
                            utilize these components.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Build
                        </h1>
                        <img src="./Build.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Build
                        </h1>
                        <p class="p_content">
                            Designing the genetic circuit was the next logical step. To construct our vision, we sought
                            the necessary genetic elements from the iGEM registry, including a promoter and ribosome
                            binding site (RBS). The circuit took shape as follows: Promoter + RBS + TatA + RBS + TatB +
                            RBS + TatC + RBS + TatE + Terminator. This intricate sequence needed to be carefully
                            assembled for it to work effectively.

                        </p>
                        <p class="p_content">
                            With the sequence in hand, we faced the challenge of ordering gene fragments. Our chosen
                            suppliers, Twist Bioscience and IDT, had specific limits on the length of sequences they
                            could synthesize. Thus, we divided the sequence accordingly. Our assembly method of choice
                            was NEB Hi-fi DNA Assembly, following a protocol that ensured each sequence had a 30-base
                            pair overlap with its neighboring fragments. Additionally, the first and last fragments had
                            overlapping sequences with the plasmid pSB1C3.
                        </p>
                        <p class="p_content">
                            We didn't leave anything to chance. Before proceeding, we rigorously verified the assembly
                            in silico using Snapgene. After thorough review and double-checking, we placed our order
                            with Twist Bioscience, setting the stage for the next phase.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Test
                        </h1>
                        <img src="./Test.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Test
                        </h1>
                        <p class="p_content">
                            While waiting for the gene fragments to arrive, we prepared the pSB1C3 plasmid from the iGEM
                            Distribution Kit 2023. This involved reviving the plasmid, establishing a primary culture,
                            and conducting plasmid isolation, ensuring that the concentration was satisfactory.
                        </p>
                        <p class="p_content">
                            Once the gene fragments arrived, we subjected them to a critical examination. Gel
                            electrophoresis was our chosen method for this task. We compared the bands to the NEB 1 kb
                            plus DNA ladder, meticulously checking for quality and integrity. With the fragments deemed
                            suitable, we moved forward to the next crucial step.
                        </p>
                        <p class="p_content">
                            We needed to linearize the plasmids to facilitate the upcoming assembly. For this, we
                            designed primers with overlapping base pairs with the first and last fragment. Both forward
                            and reverse primers were carefully selected after in-silico validation, ensuring precision
                            in our approach.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Learn
                        </h1>
                        <img src="./Learn.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Learn
                        </h1>
                        <p class="p_content">
                            The journey was not without its challenges. We initiated a gradient PCR at various
                            temperatures, ultimately observing bright bands at 62 and 64 degrees Celsius during gel
                            electrophoresis.

                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/gradient-pcr-tat.jpg"
                            alt="">

                        <p class="p_content">
                            Encouraged by these results, we proceeded with PCR at 64 degrees Celsius for gel extraction,
                            creating three reactions of 25ul each. Gel electrophoresis confirmed the presence of bright
                            bands, affirming the success of our procedure.

                        </p>
                        <p class="p_content">
                            However, we encountered unexpected obstacles during the PCR clean-up process using the NEB
                            Monarch Kit. Multiple attempts were made, optimizations were considered, such as extending
                            the elution time and increasing the incubation period, but the problem persisted. After
                            careful examination, we discovered that the PCR Clean-Up kit was nearing its expiration date
                            and had issues.

                        </p>
                        <p class="p_content">
                            Adapting to the situation, we embraced the Qiagen Gel Extraction Protocol, with valuable
                            assistance from the Protein Engineering Lab at IISER Kolkata and the Plant Biology Lab under
                            Dr. Sreeramaiah Gangappa. The collaboration proved fruitful as we successfully completed the
                            gel extraction. With concentration and 260/280 ratio within the desired range, we proceeded
                            with the NEB Hi-fi DNA Assembly.

                        </p>
                        <p class="p_content">
                            The assembly was followed by transformation into E. coli DH5alpha competent cells, and later
                            into BL21DE3 Electrocompetent cells. With successful transformation and plasmid isolation,
                            we ran further checks using gel electrophoresis, confirming our progress. Finally, we
                            performed Colony PCR for verification and sent samples for sequencing to validate our
                            efforts.

                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/cropped-tat-plasmid.jpg"
                            alt="">

                        <p class="p_content">
                            The culmination of our rigorous work and collaboration led to the creation of our assembled
                            product, a vital component for the upcoming double transformation with another circuit,
                            marking another chapter in our scientific journey.

                        </p>

                    </div>
                </div>
            </div>
        </div>
    </div>
    <!-- ---------------------------------- -->

    <div class="separator" id="4">
        <h1 class="separator_txt">
            <div>
                OMV Fusion Verification with Xanthomonas
            </div>
        </h1>
    </div>
    <div class="superStickyContainer">
        <div class="sticky_index_container"
            style="width: 10%;width: 10%;display: flex;flex-direction: column;align-items: center;">
            <div class="sticky_index2">OMV Fusion Verification with Xanthomonas</div>
            <div></div>
        </div>
        <div class="content_container" style="width: 90%;">
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Design
                        </h1>
                        <img src="./Design.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Design
                        </h1>
                        <p class="p_content">
                            One of the most essential roadblock we had was to do outer membrane vesicles released from
                            gram negative bacteria show interspecies communication. The important question we had was -
                            Is it possible that outer membrane vesicles from E.coli can fuse or interact with
                            Xanthomonas.
                            After a deep literature review we found out studies that have shown interactions between
                            Outer membrane vesicles between same species, quite few papers were shown on interactions
                            between interspecies communication.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Build
                        </h1>
                        <img src="./Build.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Build
                        </h1>
                        <p class="p_content">
                            We came across literature studies where interaction between membrane vesicles with mammalian
                            cell lines was shown. Hence, we decided to use Lipophilic Dye to stain Outer membrane
                            vesicles and study the interaction with Xanthomonas using Flow Cytometry assay and Confocal
                            microscopy.
                        </p>
                        <p class="p_content">
                            The basic design layout of the experiment was built with the help of Scientific Officer
                            Tamal Ghosh. Further, Dr Bidisha Sinha suggested us to use FM1-43 membrane staining dye to
                            stain the outer membrane of Xanthomonas and to better analyze the interaction.
                        </p>
                        <p class="p_content">
                            The Confocal microscopy experiments set the base for further future GFP localization testing
                            experiments as well and isolation and interaction studies.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Test
                        </h1>
                        <img src="./Test.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Test
                        </h1>
                        <p class="p_content">
                            <b>
                                Flow Cytometry Results
                            </b>
                            We first used Rhodamine B dye to stain Outer membrane vesicles and unstained <i>Xanthomonas
                                oryzae.</i>. Then incubated them together for 1 hour in 1:1 ratio of volume. A
                            significant amplification curve was obtained when <i>Xanthomonas oryzae.</i> were incubated
                            with OMV’s suggesting that some interaction was happening.
                            A triplicate of the following experiments was done which showed us same results hence, we
                            decided to proceed and try confocal microscopy.
                        </p>
                        <p class="p_content">
                            This cycle was crucial for us to understand that outer membrane vesicles carry the
                            possibility of potential gene transfer agent. We further went on to perform experiment with
                            GFP loaded outer membrane vesicles and positive results were obtained.
                        </p>
                        <p class="p_content">
                            <b>
                                Confocal microscopy results
                            </b>
                            The same sample preparation method was done for confocal microscopy as well but first we
                            proceeded with just DAPI as the mounting medium to stain the nucleiod of <i>Xanthomonas
                                oryzae.</i>. And the following positive results were obtained:
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/control-sample.jpg"
                            alt="">
                        <p style="text-align: center;">
                            Figure : Control Sample of <i>Xanthomonas oryzae.</i>. With DAPI staining
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/fig-3-xanhomonas-with-omv.jpg"
                            alt="">
                        <p style="text-align: center;">
                            Figure 2: Interaction of Rhodamine stained OMV’s with DAPI stained Xanthomonas, but the
                            localisation can only be observed in membranes.
                        </p>
                        <p class="p_content">
                            Then we discussed the results with various microscopy experts and Dr Bidisha Sinha helped us
                            in providing the FM1-43 dye for membrane staining which further showed the following
                            positive results.
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/engineering-success/confocal-results.png"
                            alt="">
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Learn
                        </h1>
                        <img src="./Learn.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Learn
                        </h1>
                        <p class="p_content">
                            This cycle was crucial for us to understand that outer membrane vesicles carry the
                            possibility of potential gene transfer agent. We further went on to perform experiment with
                            GFP loaded outer membrane vesicles and positive results were obtained.
                        </p>
                        <p class="p_content">
                            Imaging of just OMV’s show no signal, the aggregate of OMV’s that come together to interact
                            with Xanthomonas show amplification in signal obtained.
                        </p>
                        <p class="p_content">
                            We can improve upon this experiment by understanding the size limitation transfer of these
                            proteins and further the above protocol can be used to perform TEM imaging which can give
                            higher resolution images on the interaction taking place between OMV’s and <i>Xanthomonas
                                oryzae.</i>.
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <!-- ---------------------------------- -->

    <div class="separator" id="5">
        <h1 class="separator_txt">
            Machine Learning Model for Mutational Response
        </h1>
    </div>

    <div class="superStickyContainer">
        <div class="sticky_index_container"
            style="width: 10%;width: 10%;display: flex;flex-direction: column;align-items: center;">
            <div class="sticky_index2">
                Machine Learning Model for Mutational Response
            </div>
            <div></div>
        </div>
        <div class="content_container" style="width: 90%;">
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Design
                        </h1>
                        <img src="./Design.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Design
                        </h1>
                        <p class="p_content">
                            The idea actually came up during one of our conversations with Prof Susmita Roy when she
                            pointed out that her lab has recently developed this new ML approach for predicting
                            mutational responses in viral genome recently in light of covid outbreak and whether we can
                            incorporate that for our bacterial pathogen. After digging up some literature, we went
                            onboard with the idea. In our study, we employed a feedforward backpropagation neural
                            network characterized by a specific model architecture denoted as 3–8–8–1. This architecture
                            encompasses four distinct layers. The initial layer accommodates three input features,
                            facilitating the input data's entry into the network.Subsequently, we integrated two hidden
                            layers, each containing eight neurons, which play a pivotal role in feature transformation
                            and representation learning. The final layer, consisting of a solitary neuron, corresponds
                            to the network's output and directly addresses the prediction of our target variable.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Build
                        </h1>
                        <img src="./Build.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Build
                        </h1>
                        <p class="p_content">
                            To test out the validation of the model, we used the trained model to make predictions of a
                            validation sequence. Initially we used the Torch package in python to implement the model.
                            But the prediction did not come as expected. The probabilities were coming in order of
                            10^-08. It was time for debugging now.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Test
                        </h1>
                        <img src="./Test.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Test
                        </h1>
                        <p class="p_content">
                            After a lot of debugging and switching from Torch to Keras package, we finally tested our
                            model for predictions. And this time we were successful. The model predictions against the
                            validation file were giving probabilities of the order of 0.1. With a user defined cutoff of
                            0.15, we checked with actual predictions and found out the model predicted a total of 57
                            potential positions that can mutate in the future version of XopN. Out of these 57 predicted
                            mutations, the model predicted 9 residues that have actually mutated in the validation
                            sequence.
                        </p>
                    </div>
                </div>
            </div>
            <br>
            <div class="detailed_description_section" style="width: 100%;">
                <div class="sticky_index_container">
                    <div class="sticky_index">
                        <h1 class="h1Index">
                            Learn
                        </h1>
                        <img src="./Learn.png" alt="" srcset="">
                    </div>
                </div>
                <div class="content_container">
                    <div class="content_section" style="margin-top: 0;">
                        <h1 class="h1_content">
                            Learn
                        </h1>
                        <p class="p_content">
                            With the successful prediction of mutational sites, we learned that the features selected
                            for estimating the random mutation worked just as well in bacteria as in viruses. The
                            overall low mutational probability for bacteria might be a relic of the less amount of data
                            availability for model training as compared to the viral case. Also we learned what could be
                            the possible mutations for a given XopN sequence, which in turn can be very helpful for
                            modifying the biocontrol in near future.
                        </p>
                        <img class="image_content"
                            src="https://static.igem.wiki/teams/4781/wiki/website-images/model/ml-pipeline.png" alt="">
                    </div>
                </div>
            </div>
        </div>
    </div>




    <script>

        let nav = {
            "PROJECT": {
                "Description": "https://example.com/",
                "Implementation": "https://example.com/",
                "Contribution": "https://example.com/",
                "Graphic Design": "https://example.com/",
                "Awards": "https://example.com/"
            },
            "WET LAB": {
                "Design": "https://example.com/",
                "Experiments": "https://example.com/",
                "Results": "https://example.com/",
                "Engineering": "https://example.com/",
                "Parts": "https://example.com/",
                "Safety": "https://example.com/"
            },
            "DRY LAB": {
                "Model": "https://example.com/",
                "Software": "https://example.com/",
                "Hardware": "https://example.com/",
                "Measurement": "https://example.com/"
            },
            "HUMAN PRACTICES": {
                "Integrated Human Practices": "https://example.com/",
                "Education & Public Engagement": "https://example.com/",
                "The 6th SynBio Sense": "https://example.com/"
            },
            "TEAM": {
                "Members": "https://example.com/",
                "Collaborations": "https://example.com/",
                "Attributions": "https://example.com/"
            }
        }


        $(".clear_ul li").hover(function () {
            var isHovered = $(this).is(":hover");
            if (isHovered) {
                console.log(Object.keys(nav[this.children[0].innerHTML]));
                let dropdown = document.getElementById('dropDown_ul')
                dropdown.innerHTML = ''
                for (const key in nav[this.children[0].innerHTML]) {
                    dropdown.innerHTML += `<li><a href="${nav[this.children[0].innerHTML][key]}">${key}</a></li>`
                }
                // Object.keys(nav[this.children[0].innerHTML])
                document.getElementById('dropdown').classList.add("visible")
                document.getElementById('dropdown').classList.remove("hidden")
            } else {
                // document.getElementById('dropdown').innerHTML = "https://example.com/"
            }
        });

        function hide() {
            if (document.getElementById('dropdown').classList.contains('visible')) {
                document.getElementById('dropdown').classList.remove("visble")
                document.getElementById('dropdown').classList.add("hidden")
            }
        }

    </script>

    <script src="../old two/main.js"></script>
    <script>
        renderMathInElement(document.body, {
            delimiters: [
                { left: "$$", right: "$$", display: true },
                { left: "$", right: "$", display: true },
                { left: "\\(", right: "\\)", display: false },
                { left: "\\[", right: "\\]", display: true },
                { left: "\\begin{equation}", right: "\\end{equation}", display: true }
            ]
        });
    </script>
    <script src="../old two/table.js"></script>
    <script>
        // Sample CSV data (you can replace this with your CSV file)
        // const csvData = `Name,URL,Description
        //                 John,http://example.com/john,Software Engineer
        //                 Alice,http://example.com/alice,Web Developer
        //                 Bob,http://example.com/bob,Data Analyst`;

        // Create a new instance of the FileReader() object
        // Create a new instance of the FileReader() object
        $(document).ready(function () {
            $.ajax({
                type: "GET",
                url: "../old two/csv/engineeringSuccess_pt2.csv",
                dataType: "text",
                success: function (data) { renderTablewithoutlinks(data, 'engineeringSuccess_pt2') }
            });
        });$(document).ready(function () {
            $.ajax({
                type: "GET",
                url: "../old two/csv/engineeringSuccess_pt2.csv",
                dataType: "text",
                success: function (data) { renderTablewithoutlinks(data, 'engineeringSuccess_pt2') }
            });
        });


        // let csvData2 = 
    </script>
    <script>
        // $(document).ready(function (e) {
        //     $('rect').on('focus', function (e) {
        //         e.preventDefault();
        //         alert($(this).attr('id'))
        //         // $('.selection p').html($(this).attr('class'));
        //     });

        // });

    </script>
    <!-- <script src="../old two/indexMaker.js"></script> -->
    <script src="../old two/grassAnimation.js"></script>
</body>

</html>